### PR TITLE
REGD-132 fix dark mode and summary table

### PR DESCRIPTION
**Pull request by mingjiecn**
1. A bug is fixed in GDS so the summary endpoint will not redirect to search endpoint if there is error in query. So for regulome UI, the summary table should be shown even if there is only one variant returned.
2. Remove the unnecessary style and use the style defined in config and global css to make sure the dark mode is working and the style is consistent.

### CODE REVIEW

File: components/body-map.js [hunk 1]
```diff
@@ -149,8 +149,10 @@ export function BodyMap({
                   const borderStyle = isSelected
                     ? "border-solid border-2 border-brand"
                     : "";
-                  const textColor = isDisabled ? "text-slate-400" : "";
-                  const highlight = isHighlighted ? "bg-green-200" : "";
+                  const textColor = isDisabled
+                    ? "text-button-selected-disabled"
+                    : "";
```
Comment (forresttanaka) [inline components/body-map.js]: Another option is to use `disabled:text-button-selected-disabled` without a condition — it’s always a class of the button element whether it’s disabled or not. Tailwind CSS only uses classes starting with `disabled:` if the element is actually disabled.

File: components/genome-browser.js [hunk 2]
```diff
@@ -334,7 +334,7 @@ export default function GenomeBrowser({ files, assembly, coordinates }) {
     return (
       <button className="reset-browser-button">
         <ArrowUturnLeftIcon className="h-5" />
-        <span className="reset-title">Reset to query variant label</span>
+        <span>Reset to query variant label</span>
```
Comment (forresttanaka) [inline components/genome-browser.js:337]: The `<span>` elements probably make no difference, depending on how the reset-browser-button class is defined.

**Review by forresttanaka — APPROVED** [review]
Just a couple optional comments.

File: components/body-map.js (hunk 1 above)
Comment (mingjiecn) [reply components/body-map.js]: Thanks for the tips!

**Pull request merged**